btcjson/walletsvrresults.go

```diff
@@ -173,3 +173,18 @@ type GetBalancesResult struct {
 	Mine      BalanceDetailsResult  `json:"mine"`
 	WatchOnly *BalanceDetailsResult `json:"watchonly"`
 }
+
+// WalletCreateFundedPsbtResult models the data returned from the
+// walletcreatefundedpsbtresult command.
+type WalletCreateFundedPsbtResult struct {
+	Psbt      string  `json:"psbt"`
```

Comment:
Not so familiar with psbt transaction format, but can we use `*wire.MsgTx` here just like in [`FundRawTransactionResult`](https://github.com/btcsuite/btcd/blob/1db1b6f8217f5b489505c5c7e5235b53a7484779/btcjson/chainsvrresults.go#L695-L700). Will require special unmarshalling logic. See an example [here](https://github.com/btcsuite/btcd/blob/1db1b6f8217f5b489505c5c7e5235b53a7484779/btcjson/chainsvrresults.go#L702-L732).

Reply:
The PSBT transaction format is different from the wire tx format. It's described in [BIP 174](https://github.com/bitcoin/bips/blob/master/bip-0174.mediawiki) as consisting of key-value maps.